MigrationScript/AmsOperationsHelper.ps1

```diff
@@ -286,8 +285,63 @@ function GetAmsV2ManagedFunc([string]$subscriptionId, [string]$resourceGroup, [s
 		$logger.LogError("GetAmsV2ManagedKv : Failed with error: ($($GetProviderErrorMsg.error.code)))", "500", "");
     }
 
-	return @{
-		Response = $managedFuncName
-		Error = $GetProviderErrorMsg
-	}
+	return $managedFuncName;
+}
+
+
+<#
+.SYNOPSIS
+Function to get the deployment status for Provider.
+
+.PARAMETER subscriptionId
+SubscriptionId of the AMS v1 Monitor.
+
+.PARAMETER resourceGroup
+Resource Group Name of the AMS v1 Monitor.
+
+.PARAMETER monitorName
+AMS v1 Monitor Name.
+
+.PARAMETER providerName
+Provider name.
+
+.PARAMETER logger
+logger object. 
+
+.EXAMPLE
+. $PSScriptRoot\ConsoleLogger.ps1
+$logger = New-Object ConsoleLogger
+GetAmsV1ProviderStatus -subscriptionId $subscriptionId -resourceGroup $resourceGroupName -monitorName $monitorName -providerName $providerName -logger $logger;
+#>
+function GetAmsV1ProviderStatus([string]$subscriptionId, [string]$resourceGroup, [string]$monitorName, [string]$providerName, $logger)
+{
+    $rawToken = Get-AzAccessToken -ResourceTypeName Arm
+    $armToken = $rawToken.Token
+	$v2ApiVersion = "2020-02-07-preview";
+
+    $headers = @{
+        "Content-Type" = "application/json"
+        "Authorization" = "Bearer $armToken"
+    }
+
+    [string]$url = $url = "https://management.azure.com/"
+	[string]$subscriptionParams = "subscriptions/" + $subscriptionId;
+	[string]$rgParams = "/resourceGroups/" + $resourceGroup;
+	[string]$providerParams = "/providers/Microsoft.HanaOnAzure/sapMonitors/" + $monitorName + "/providerInstances/" + $providerName + "?api-version=" + $v2ApiVersion;
+	$url = $url + $subscriptionParams + $rgParams + $providerParams;
+	[string]$provisiongState = "";
+	$logger.LogInfo("Making Get Provider call with $url")
```

Comment:
missed ; here ?

Reply:
Added semicolon in the latest commit, not sure thy this is not coming up here.